sheffield-Armin Nouri-Wireframe-week one

ArminNouri98 opened the pull request:
pseudo-class. Apply styling through an external CSS file.

<!--

You must title your PR like this:

COHORT_NAME | FIRST_NAME LAST_NAME | REPO_NAME | WEEK

For example,

NW4 | Carol Owen | HTML-CSS-Module | Week1

Complete the task list below this message.
If your PR is rejected, check the task list.

-->

## Learners, PR Template

Self checklist

- [x] I have committed my files one by one, on purpose, and for a reason
- [x] I have titled my PR with COHORT_NAME | FIRST_NAME LAST_NAME | REPO_NAME | WEEK 
- [x] I have tested my changes
- [x] My changes follow the [style guide](https://syllabus.codeyourfuture.io/guides/code-style-guide/)
- [x] My changes meet the [requirements](./README.md) of this task

## Changelist

Briefly explain your PR.

## Questions

Ask any questions you have for your reviewer.

I don’t know I have done right or no i need change?!

ArminNouri98 committed:
Create a form control that collects information about the customer, including their name, address, and details about the selected T-shirt such as size, color, and shipping date.

ArminNouri98 committed:
Add background color, title, and header styling to the page

ArminNouri98 committed:
Implement consistent color scheme for links and text. Structure webpage using semantic HTML tags. Add header with title and description. Create three articles, each with a title, summary, and link. Fix footer to the bottom of the viewport. Style the first article using the
pseudo-class. Apply styling through an external CSS file.

cjyuan commented:
1. Does the layout of your page resemble the one shown in the wireframe?  (The wireframe has a very specific layout)
2. The page header does not have a clear title and description. 
3. Each article should include a title, summary, and a link.
4. The spec mentions the page should contain certain content.
5. The footer is not fixed at the bottom of the browser window. 
6. The article tag is not used correctly. If there are three articles in the page, then there should be only three article elements.
7. Some of the CSS styles are not being used; the selector does not select any element in your page. If they are not used, you should remove them.

ArminNouri98 committed:
done changing :
Does the layout of your page resemble the one shown in the wireframe? (The wireframe has a very specific layout)
The page header does not have a clear title and description.
Each article should include a title, summary, and a link.

ArminNouri98 committed:
footer is fixed

ArminNouri98 committed:
add three article elements, change class article make one in each article card

ArminNouri98 committed:
I edit css as much as I can and clean up it

ArminNouri98 committed:
add first child and edit README

ArminNouri98 committed:
fix bug of first child in css for first card

ArminNouri98 committed:
edit html and add more explanation

ArminNouri98 committed:
change a topic of Articles to git

ArminNouri98 commented:
I have updated the topics and images in Git

I implemented the :first-child pseudo-class for styling the cards of the articles.

cjyuan commented:
All looks good now! 

Abut your media query: `@media only screen and (min-width: 150px)`
Have you searched what are the typical viewport width of various devices (mobile phone, touch pad, etc)?
I think almost all devices will qualify the `min-width: 150px` condition.

ArminNouri98 commented:
I used it to sort articles while checking on my phone because it provides a clean and consistent view. it's first time I useing this  :
https://www.w3schools.com/css/css3_mediaqueries_ex.asp
